test/toolkits/style_transfer/test_style_transfer_data_iterator.cxx

```diff
@@ -7,26 +7,23 @@
 
 #define BOOST_TEST_MODULE
 
+#include <string>
+#include <utils.hpp>
```

Comment:
Did you try <toolkits/style_transfer/utils.hpp>?

If that path also exists on the non-test side, maybe give it a different name…

Reply:
Path wasn't directly available to us. Fixed this however by putting the include_directories in the top CMakeLists under the test directory.

Reply:
What do you mean by "path wasn't directly available to us?"

@hoytak @znation How is test code supposed to import headers defined under `/test`? Or do we just not support headers there? Does test utility code need to be in `/src`?

Reply:
I think I've done this before by using relative include paths (like `"../..."`). I think `<test/...>` should also work though I haven't tried it.

Reply:
Sorry to be late to the game here -- but yes, relative imports in the test directory are a good idea.  We should not risk src/ code picking up things from the test/ directory.